bin/svg2png-cli.js

```diff
@@ -33,6 +37,13 @@ const argv = yargs
 
 const input = fs.readFileSync(argv._[0]);
 const output = svg2png.sync(input, { width: argv.width, height: argv.height });
-
 const outputFilename = argv.output || path.basename(argv._[0], ".svg") + ".png";
-fs.writeFileSync(outputFilename, output, { flag: "wx" });
+const writeFlag = ( argv.overwrite ) ? 'w' : 'wx';
+
+try {
+    if ( fs.statSync(outputFilename) && argv.overwrite === false ) throw 'File already exists.';
```

Comment:
Thanks for the feedback, learned a few things from it.  Student starting out, so had too google the race condition thing.  Anyway, noted and applied the changes.